package/MDAnalysis/topology/MOL2Parser.py

```diff
@@ -97,6 +97,8 @@ class MOL2Parser(TopologyReaderBase):
        Bonds attribute is not added if no bonds are present in MOL2 file
     .. versionchanged: 2.0.0
        Parse elements from atom types.
+    .. versionchanged:: 2.2.0
+       Optional columns will be assigned with default values when not provided
```

Comment:
Explicit the chosen default values here as well, so they do show up in the docs.

Reply:
@RMeli I added how to deal with optional fields in class docstring, should I added it in versionchanged tag as well?

Reply:
@orbeckst suggestion to add it to the docstring is much better, I think the `versionchanged` does not need this information.